### PR TITLE
Hides cancel button on demand

### PR DESCRIPTION
**Pull request by pepsin**

### CODE REVIEW

File: Source/Configuration/YPImagePickerConfiguration.swift [hunk 1]
```diff
@@ -74,6 +74,9 @@ public struct YPImagePickerConfiguration {
     
     /// Adds a Overlay View to the camera
     public var overlayView: UIView?
+	
+	/// Defines if the navigation bar cancel button should be hidden when showing the picker. Default is false
+	public var hidesCancelButton = false
```
Comment (NikKovIos) [inline Source/Configuration/YPImagePickerConfiguration.swift:79]: Fix the indentation.

File: Source/Helpers/Extensions/PHCachingImageManager+Extensions.swift [hunk 2]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2018 Yummypets. All rights reserved.
 //
 
-import Foundation
+import UIKit.UIImage
```
Comment (NikKovIos) [inline Source/Helpers/Extensions/PHCachingImageManager+Extensions.swift:9]: Not necessary change. Remove it.

File: Source/YPPickerVC.swift [hunk 3]
```diff
@@ -263,11 +263,13 @@ open class YPPickerVC: YPBottomPager, YPBottomPagerDelegate {
...
-                                                           style: .plain,
-                                                           target: self,
-                                                           action: #selector(close))
+		if !YPConfig.hidesCancelButton {
+	        // Update Nav Bar state.
+	        navigationItem.leftBarButtonItem = UIBarButtonItem(title: YPConfig.wordings.cancel,
+	                                                           style: .plain,
+	                                                           target: self,
+	                                                           action: #selector(close))
+		}
```
Comment (NikKovIos) [inline Source/YPPickerVC.swift:272]: Fix the indentation.

File: Source/Helpers/YPPhotoSaver.swift [hunk 4]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2016 Yummypets. All rights reserved.
 //
 
-import Foundation
+import UIKit.UIImage
```
Comment (NikKovIos) [inline Source/Helpers/YPPhotoSaver.swift:9]: Not necessary change. Remove.

**Review by NikKovIos — CHANGES_REQUESTED** [review]

**Comment by s4cha** [comment]
@pepsin Congrats for your first contribution! I guess this is for the case where you want the user to select a picture mo matter what and prevent it from going back. Looks like a valid use case to me :)

**Comment by s4cha** [comment]
Could you please fix the conflicts ?

**Comment by s4cha** [comment]
I just added your request, lots of things happened in between which made it hard to merge so I directly updated master with your changes.

**Pull request closed without merging**